### PR TITLE
White Shoes

### PR DESCRIPTION
**Pull request by LordWalnut**
Adds white shoes to the game

### CODE REVIEW

File: code/modules/clothing/shoes/colour.js [hunk 1]
```diff
@@ -29,5 +29,20 @@ module.exports.templates = {
...
+				},
+				"Examine": {
+					desc: "A pair of white shoes."
+				}
+			},
+			name: "white shoes",
+			icon_state: "white"
+			}
+		}
+	};
```
Comment (monster860) [inline code/modules/clothing/shoes/colour.js]: the indentation is fucked up, you need to remove one level of tab from these 3 `}` lines.

File: code/modules/clothing/shoes/colour.js [hunk 2]
```diff
@@ -29,5 +29,20 @@ module.exports.templates = {
 			name: "brown shoes",
 			icon_state: "brown"
 		}
-	}
-};
+	},"shoes_white": {
```
Comment (monster860) [inline code/modules/clothing/shoes/colour.js]: Put a newline between the `},` and `"shoes_white"`

**Comment by LordWalnut** [comment]
I should probably mention that I've fixed the issues, I'm dumb and didn't put that in the description

File: code/modules/clothing/shoes/colour.js [hunk 3]
```diff
@@ -29,5 +29,23 @@ module.exports.templates = {
...
+				}
+			},
+			name: "white shoes",
+			icon_state: "white"
+
+		}
 	}
 };
+
+
```
Comment (monster860) [inline code/modules/clothing/shoes/colour.js]: oi get rid of this line with 2 tabs.

**Pull request merged**